src/NumericalAlgorithms/DiscontinuousGalerkin/NumericalFluxes/Hll.hpp

```diff
@@ -100,29 +98,32 @@ struct Hll {
       expand_pack(
           (get<NormalDotFluxTags>(*packaged_data) = n_dot_f_to_package)...);
 
-      get<MinSignalSpeed>(*packaged_data) = make_with_value<Scalar<DataVector>>(
-          characteristic_speeds[0],
-          std::numeric_limits<double>::signaling_NaN());
-      get<MaxSignalSpeed>(*packaged_data) = make_with_value<Scalar<DataVector>>(
-          characteristic_speeds[0],
-          std::numeric_limits<double>::signaling_NaN());
+      get<FirstSpeedEstimate>(*packaged_data) =
+          make_with_value<Scalar<DataVector>>(
+              characteristic_speeds[0],
+              std::numeric_limits<double>::signaling_NaN());
+      get<SecondSpeedEstimate>(*packaged_data) =
+          make_with_value<Scalar<DataVector>>(
+              characteristic_speeds[0],
+              std::numeric_limits<double>::signaling_NaN());
 
-      // This finds the min and max characteristic speeds at each grid point,
-      // which are used as estimates of the min and max signal speeds.
+      // When packaging interior data, FirstSpeedEstimate and
+      // SecondSpeedEstimate will hold the min and max char speeds,
+      // respectively. On the other hand, when packaging exterior data, the
+      // characteristic speeds will be computed along *minus* the exterior
+      // normal, so FirstSpeedEstimate will hold *minus* the max speed, while
+      // SecondSpeedEstimate will store *minus* the min speed.
```

Comment:
For discussion — I feel that the names "first speed" and "second speed" obscure the fact that *some* structure remains in the contents of these variables. Would it be fair to say that the first speed is the "largest ingoing" or perhaps the "largest antiparallel to the normal" speed, and, similarly, that the second is the "largest outgoing" or "largest parallel to the normal" speed? If we can think of a name that reflects the structure in this way, I would rather prefer it over the very generic (but consequently low in information) names  "first speed" and "second speed".

Reply:
I'm not committed to these names so I'm open to any changes. (If so, HLLC should also be updated.) Maybe @wthrowe can suggest better names, since he fixed the communicated speeds in the first place.

Reply:
Let's go with FirstSpeed -> LargestIngoing and SecondSpeed -> LargestOutgoing